entities/params/sendcampaign.go

```diff
@@ -0,0 +1,13 @@
+package params
+
+import "strings"
+
+type SendCampaignParams struct {
+	Ids      []int64 `form:"segment_id[]" validate:"required"`
```

Comment:
I'm trying some implementation for validating slice. but we should discuss this @FilipNikolovski

Reply:
Check out https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Dive it allows you to dive into the slice and apply a validation on each element.